packages/manager/src/utilities/getEventsActionLink.ts

```diff
@@ -102,6 +102,11 @@ export default (
           return `/nodebalancers/${id}/summary`;
       }
 
+    case 'database':
+      const engine =
+        entity?.url.split('/v4/databases/')[1].split('/')[0] || 'mysql';
```

Comment:
This is questionable. Any suggestions?

Reply:
What about using a regex?

```suggestion
        entity?.url.match(/(mysql|postgresql|mongodb|redis)/i)[0];
```

Reply:
Looks nicer, but requires us to update this regex every time we add a new database engine. May still use it because it will probably be a long time until we add anything other than mysql, postgres, mongo, and redis.

Reply:
Good point, what about

```typescript
entity?.url.match(/databases\/(\w*)\/instances/i)?.[1]
```